R/tramitacoes.R

```diff
@@ -60,53 +20,49 @@ fetch_tramitacao_senado <- function(proposicao_id, normalized=FALSE) {
                     sigla_local = origem_tramitacao_local_sigla_local,
                     id_situacao,
                     descricao_situacao = situacao_descricao_situacao)
-  }
-  
-  proposicao_tramitacoes_df
-}
-
-#' @title Renomeia as colunas do dataframe de movimentação no Senado
-#' @description Renomeia as colunas do dataframe de movimentação no Senado usando o padrão
-#' de underscore e letras minúsculas
-#' @param df Dataframe da votação no Senado
-#' @return Dataframe com as colunas renomeadas
-#' @export
-rename_tramitacao_df <- function(df) {
-  new_names = names(df) %>%
-    to_underscore() %>%
-    stringr::str_replace(
-      "identificacao_tramitacao_|
-      identificacao_tramitacao_origem_tramitacao_local_|
-      identificacao_tramitacao_destino_tramitacao_local_|
-      identificacao_tramitacao_situacao_",
-      ""
-    )
-  
-  names(df) <- new_names
-  
-  df
 }
 
 #' @title Baixa os dados da tramitação de um Projeto de Lei
 #' @description Retorna dataframe com os dados da tramitação de uma proposição no Congresso
 #' @param id ID de uma proposição na sua respectiva casa
 #' @param casa Casa onde a proposição está tramitando
-#' @param normalized whether or not the output dataframe should be normalized (have the same format and column names for every house)
-#' @return Dataframe com os dados da tramitação de uma proposição no Congresso
+#' @return Dataframe normalizado com os dados da tramitação de uma proposição no Congresso
 #' @examples
-#' fetch_tramitacao(91341,'senado')
+#' fetch_tramitacao(91341, "senado")
 #' @export
-fetch_tramitacao <- function(id, casa, normalized=FALSE) {
+fetch_tramitacao <- function(id, casa) {
   casa <- tolower(casa)
-  if (casa == 'camara') {
-    fetch_tramitacao_camara(id, normalized)
-  } else if (casa == 'senado') {
-    fetch_tramitacao_senado(id, normalized)
+  if (casa == "camara") {
+    fetch_tramitacao_camara(id)
+  } else if (casa == "senado") {
+    fetch_tramitacao_senado(id)
   } else {
-    print('Parâmetro "casa" não identificado.')
+    print("Parâmetro 'casa' não identificado.")
   }
 }
 
+#' @title Baixa os dados da tramitação da Câmara
+#' @description Retorna dataframe com os dados da tramitação de uma proposição da Camara
+#' @param bill_id ID de uma proposição na Camara
+#' @return Dataframe com os dados da tramitação de uma proposição da Camara
+#' @examples
+#' fetch_tramitacao_camara(2121442, TRUE)
+fetch_tramitacao_camara <- function(bill_id) {
+  rcongresso::fetch_tramitacao_camara(bill_id) %>%
+    rcongresso::rename_df_columns() %>%
```

Comment:
Error: 'rename_df_columns' is not an exported object from 'namespace:rcongresso'

Reply:
Fiz um PR que deve arrumar isso:
https://github.com/andresmrm/rcongresso/pull/27